hudi-flink/src/main/java/org/apache/hudi/sink/compact/HoodieFlinkCompactor.java

```diff
@@ -18,9 +18,16 @@
 
 package org.apache.hudi.sink.compact;
 
+import org.apache.flink.api.common.typeinfo.TypeInformation;
```

Comment:
the import order is wrong here. The checkstyle should fail. please follow hudi -> 3rd party -> java/scala -> static order